.appveyor.yml

```diff
@@ -72,13 +69,9 @@ before_build:
          Write-Host "Qt binaries already present.";
       }
 - cmd: python build_msvc\msvc-autogen.py
-- ps:  Start-Process clcache-server
-- ps:  fsutil behavior set disablelastaccess 0 # Enable Access time feature on Windows (for clcache)
 build_script:
-- cmd: msbuild /p:TrackFileAccess=false /p:CLToolExe=clcache.exe build_msvc\bitcoin.sln /m /v:q /nologo
+- cmd: msbuild /p:TrackFileAccess=false build_msvc\bitcoin.sln /m /v:q /nologo
 after_build:
-- ps:  fsutil behavior set disablelastaccess 1 # Disable Access time feature on Windows (better performance)
```

Comment:
Removing this line seems unrelated.

Reply:
See this build here: https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/32194931

So with this line added, it slowed down the build. Or at least didn't speed it up, so I don't feel comfortable keeping it.

Test code should be minimal, easy to read and easy to write. Having a line say "better performance" without any data to back up the claim doesn't help.